tests/unit/test_package.py

```diff
@@ -37,6 +37,7 @@ def test_deploy_is_called(self, mock_put, mock_sudo):
     def test_rpm_install(self, mock_sudo):
         env.host = 'any_host'
         env.nodeps = False
+        env.java8_home = None
```

Comment:
you will probably be able to get rid of these changes with the above suggested change